package.json

```diff
@@ -23,6 +23,10 @@
   "peerDependencies": {
     "hardhat": "^2.0.0"
   },
+  "dependencies": {
+    "@ethersproject/abi": "^5.5.0",
```

Comment:
These should be dev dependencies.

Reply:
Dev dependencies are _only_ installed in a local clone of the repository when `yarn install` is run.  These packages need to be included in end users' projects.  Whether `hardhat-abi-exporter` itself is installed as a dev dependency or normal dependency, only its normal dependencies will be installed along with it.